### ISSUE TITLE
getPayload return wrong value

### ISSUE THREAD

**Issue by 3alisaki** [versatica/rtp.js#26]
the getPayload methid of RtpPacket is returning the same data view that was passed to RtpPacket constructor, what i know is that it should return rtp payload without headers, pleas fix that.

**Comment by ibc** [comment]
No. This is correct as it is. `packet.getPayload()` returns a `DataView` that is NOT the same as the `DataView` given to the constructor. And you must always use `dataView.byteOffset` and `dataView.byteLength` because indeed both `DataViews` may use the same `ArrayBuffer` instance internally, but byte offsets and byte length of the views are different. So no bug here, it's done as intended and as documented.
